docs/mine-hnt/miner-cli/minercli.mdx

```diff
@@ -8,9 +8,73 @@ slug: /mine-hnt/miner-cli/minercli
 import Tabs from "@theme/Tabs";
 import TabItem from "@theme/TabItem";
 
----
+# Introduction
+The `miner` command controls validators and hotspot miners it is also the core software that runs all aspects of the blockchain, like adding txns, and validating blocks while in a consensus group. The CLI is used to troubleshoot and maintain these devices. 
+
+You can always get more detailed help in the `#blockchain-development` channel in [Discord](https://discord.com/channels/404106811252408320/730245219974381708).
+
+:::info
+
+The github repository to review the code for the miner command is found [here](https://github.com/helium/miner) Note the differing tags for validators and hotspots.
+
+:::
+
+## Example page syntax
+
+The complexity of all the different branches of the command is located on sperate pages. One for each major section of the command to keep the page length small. A navgation menu will be on the right hand column to navigate through each sub command. There is a Tab for the Command "inputs" and example outputs of each command are also on an individual tab.
```

Comment:
"The complexity of all the different branches of the command is located on sperate pages. One...small" -> 
"The full usage details for all `miner` commands are located on dedicated command pages. There is one for each top-level command to keep the docs focused."

Reply:
fixed